qiskit/providers/ibmq/ibmqsingleprovider.py

```diff
@@ -90,12 +91,14 @@ def _authenticate(self, credentials):
             if version_info['new_api'] and 'api-auth' in version_info:
                 return IBMQClient(api_token=credentials.token,
                                   auth_url=credentials.url,
-                                  proxies=proxies)
+                                  verify=verify, proxies=proxies)
             else:
                 # Prepare the config_dict for IBMQConnector.
                 config_dict = {
                     'url': credentials.url,
                 }
+                if credentials.websocket_url:
```

Comment:
This line seems to be a result of a wrong merge - can you check?

Reply:
Should this block be removed?

Reply:
Only the websocket-related two lines - as they are not in `master`, and they seemed to be an unintended side effect of a previous merge on your branch.